test/testingHelpers.js

```diff
@@ -112,9 +112,8 @@ const mockOAuth2Client = {
       throw new Error("Couldn't parse token")
     }
 
-    // replicates error thrown by Google's OAuth2Client for an expired token
     if (Date.now() / 1000 > payload.exp) {
-      throw new Error(`Token used too late, ${Date.now()} > ${payload.exp}: ${JSON.stringify(payload)}`)
+      throw new Error('Token used too late')
```

Comment:
I'm happy with this shorter error message. Just curious why you took out the extra detail.

Reply:
I took out the extra detail as it made testing for this error message difficult; there ended up being a chance that a second passed between the error being generated, and the test case being checked (see my first commit for this PR; one check passed, the other failed). I thought the solution was either to stub Date.now(), or remove the specificity of the message. Thoughts on another workaround?

Reply:
Thanks for explaining. That makes sense to me. In fact, there is an argument to be made about having API error message have less detail. The more detail that you give, the more information hackers have about how your system works. So I'd say just leave it as the shorter version.

Reply:
Cool!